files/__addonLocation__/package.json

```diff
@@ -33,33 +36,19 @@
     <% if (typescript) { %>"@babel/preset-typescript": "^7.18.6"<% } else { %>"@babel/eslint-parser": "^7.19.1"<% } %>,
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.20.13",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
```

Comment:
this will go away with this PR: https://github.com/embroider-build/addon-blueprint/pull/136